documentation/oracle/dvm_interfaces.md

```diff
@@ -153,7 +153,8 @@ Reveals a vote that the voter committed to during the commit period.
 This method takes the `identifier` and `time` to identify the price request.
 
 So the reveal can be verified on-chain, the voter must also provide the `price` and `salt` that they used to compute
-the `hash` that they passed to `commitVote`.
+the `hash` that they passed to `commitVote`. Since transaction data is public, the salt will be revealed with the vote. While this is the system’s expected behavior,
```

Comment:
nit: maybe we should also put this warning in the commitVote section.

Reply:
good idea done.